docs/whatsnew/4.3.rst

```diff
@@ -38,6 +38,18 @@ made much more precise (down to the milli-arcsec level).  To help this, the
 Furthermore, by ensuring expensive calculations of the precession-nutation
 matrix are only done once, the transformations have been sped up considerably.
 
+.. _whatsnew-4.3-threadsafe:
+
+Improvements in making Astropy threadsafe
+=========================================
+
+To help use of Astropy in software for the Square Kilometer Array, an effort
```

Comment:
```suggestion
To help use of Astropy in software for the Square Kilometre Array, an effort
```

Reply:
I wouldn't say it's just for the SKA though. I work on the MeerKAT radio telescope where we're also going to want these fixes. Maybe something like "To help use of Astropy in multi-threaded environments (for example, with Dask), ..."

Reply:
Thanks, done